source/modules/MEI.shared.xml

```diff
@@ -3559,6 +3565,40 @@
         C to C♯ and one from C to D♭.</p>
     </remarks>
   </classSpec>
+  <classSpec ident="att.tuning.log" module="MEI.shared" type="atts">
+    <desc xml:lang="en">Logical domain attributes for tuning.</desc>
+    <attList>
+      <attDef ident="tuning.standard" usage="opt">
+        <desc xml:lang="en">Describes the tuning standard used.</desc>
+        <datatype>
+          <rng:data type="NMTOKEN"/>
+        </datatype>
+        <valList type="semi">
+          <valItem ident="guitar.standard">
+            <desc xml:lang="en">Standard tuning for current guitars. The courses are tuned to E2 A2 D3 G3 B3 E4.</desc>
+          </valItem>
+          <valItem ident="guitar.drop.D">
+            <desc xml:lang="en">Drop D tuning for guitars. The lowest course is tuned down to D, while all other courses are kept to their regular pitches. D2 A2 D3 G3 B3 E4.</desc>
+          </valItem>
+          <valItem ident="guitar.open.D">
+            <desc xml:lang="en">Open D tuning for guitars. D2 A2 D3 F3s A3 D4.</desc>
```

Comment:
It would be good to keep the course specifications consistent. In your examples for `tab.courses` you have `ef4` (lower pitch-accid-oct), but here you have `F3s` (upper pitch-oct-accid).

Reply:
`tab.courses` has been deprecated and therefore taken out of documentation, so the inconsistency is resolved